build_library/build_image_util.sh

```diff
@@ -298,6 +298,9 @@ finish_image() {
     sudo fstrim "${root_fs_dir}/usr" || true
   fi
 
+  # Build the selinux policy
+  sudo chroot ${root_fs_dir} bash -c "cd /usr/share/selinux/mcs; semodule -i *.pp"
```

Comment:
If this isn't practical to do in a cross friendly manner right now it'll need to be wrapped in something like `if [[ "$BOARD" = amd64* ]]` to keep arm happy.